atest/robot/libdoc/resource_file.robot

```diff
@@ -100,6 +100,9 @@ Keyword tags
     Keyword Tags Should Be          8    \${3}   a    b
     Keyword Tags Should Be          9    bar    dar    foo
 
+Resource tags
+    Resource Tags Should Be          ?!?!??    Has    kw4    tags    \${3}   a    b    bar    dar    foo
+
```

Comment:
Would expect that test in line 43 or so. Where the global Resource information are checked not somewhere here where Keyword specifics are done.

Reply:
Makes sense, done.